pages/linux/aurvote.md

```diff
@@ -0,0 +1,25 @@
+# aurvote
+
+> Vote for packages in the Arch User Repository.
```

Comment:
```suggestion
> Submit votes for packages in the Arch User Repository.
```

Reply:
Why? I like my version.

Reply:
Then keep it.

This suggestion isn't really that meaningful tbh, feel free to resolve.